packages/manager/apps/hpc-vmware-managed-vcd/src/data/hooks/useManagedVcdOrganization.ts

```diff
@@ -16,11 +21,17 @@ const getVcdOrganizationBackupQueryKey = (id: string) => [
   `${getVcdOrganizationQueryKey(id)}/backup`,
 ];
 
-const useManagedVcdOrganization = (id: string) => {
+const useManagedVcdOrganization = ({
+  id,
+  refetchInterval,
+  refetchOnWindowFocus,
+}: IUseManagedVcdOrganization) => {
   return useQuery<ApiResponse<IVcdOrganization>, ApiError>({
     queryKey: getVcdOrganizationQueryKey(id),
     queryFn: () => getVcdOrganization(id),
     retry: false,
+    refetchInterval,
+    refetchOnWindowFocus,
     ...{
       keepPreviousData: true,
     },
```

Comment:
```suggestion
   keepPreviousData: true,
```

Reply:
Thanks for pointing this out, I've updated it according to the tanstack-query v5 docs (keepPreviousData option removed)